### PR TITLE
LPS-189073 Do not duplicate constants

### PR DESCRIPTION
**Pull request by robertoDiaz**
follow up requested by brian

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java [hunk 1]
```diff
@@ -120,9 +121,13 @@ public Redirect getRedirect(
 		if (pos != -1) {
 			String friendlyURL = path.substring(pos);
 
-			if (friendlyURL.startsWith(_PATH_DOCUMENTS)) {
-				String fileEntryFriendlyURL = friendlyURL.substring(
-					_PATH_DOCUMENTS.length() - 1);
+			if (friendlyURL.startsWith(
+					FriendlyURLResolverConstants.URL_PREFIX_FILE_ENTRY)) {
+
+				int length =
```
[the remark this answers is not in the document]
Comment (robertoDiaz) [reply modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java:127]: SF does not agree with you :D

File: portal-kernel/src/com/liferay/portal/kernel/portlet/constants/FriendlyURLResolverConstants.java [hunk 2]
```diff
@@ -21,6 +21,8 @@
  */
 public class FriendlyURLResolverConstants {
 
+	public static final String URL_PREFIX_FILE_ENTRY = "/documents/d/";
```
[the remark this answers is not in the document]
Comment (robertoDiaz) [reply portal-kernel/src/com/liferay/portal/kernel/portlet/constants/FriendlyURLResolverConstants.java:24]: I'll regen it, thanks!

**Comment by robertoDiaz** [comment]
ci:test:relevant

File: modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (AliciaGarciaGarcia) [reply modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java:127]: 😡

[the remark this answers is not in the document]
Comment (robertoDiaz) [reply modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java:127]: 🌚

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by Jeremy-Ch** [comment]
The failures are unrelated.

**Comment by Jeremy-Ch** [comment]
ci:test:relevant

**Comment by WYZ-Issac** [comment]
ci:test:relevant

**Comment by WYZ-Issac** [comment]
ci:test:relevant

**Comment by WYZ-Issac** [comment]
The failures are unrelated. I will forward this directly.

**Comment by WYZ-Issac** [comment]
ci:forward:force

**Pull request closed without merging**